tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,136 @@
+        pdt.assert_almost_equal(actual, expected)
+
+    def test_levels(self, plasma, reference):
+        actual = pd.DataFrame(plasma.levels)
+        expected = reference.select(os.path.join('plasma', 'levels'))
+        pdt.assert_almost_equal(actual, expected)
+
+    scalars_properties = ['time_explosion', 'link_t_rad_t_electron']
+
+    @pytest.mark.parametrize("attr", scalars_properties)
+    def test_scalars_properties(self, plasma, reference, attr):
+        actual = getattr(plasma, attr)
+        if hasattr(actual, 'cgs'):
+            actual = actual.cgs.value
+        expected = reference.select(os.path.join(
+            'plasma', 'scalars'))[attr]
+        pdt.assert_almost_equal(actual, expected)
+
+    def test_helium_treatment(self, plasma, reference):
+        actual = plasma.helium_treatment
+        expected = reference.select(os.path.join(
+            'plasma', 'scalars'))['helium_treatment']
+        assert actual == expected
+
+
+class TestLTEPlasma(BasePlasmaTest):
+
+    @pytest.fixture(scope="class")
+    def data(self, tardis_ref_path):
+        plasma_data = {}
+        plasma_data['reference_file_path'] = os.path.join(
+            tardis_ref_path, 'plasma_reference', 'plasma_lte_reference.h5')
+        config_path = os.path.join(
+            'tardis', 'plasma', 'tests', 'data', 'plasma_test_config_lte.yml')
+        plasma_data['config'] = Configuration.from_yaml(config_path)
+        return plasma_data
+
+
+class TestNLTEPlasma(BasePlasmaTest):
+
+    @pytest.fixture(scope="class")
+    def data(self, tardis_ref_path):
+        plasma_data = {}
+        plasma_data['reference_file_path'] = os.path.join(
+            tardis_ref_path, 'plasma_reference', 'plasma_nlte_reference.h5')
+        config_path = os.path.join(
+            'tardis', 'plasma', 'tests', 'data', 'plasma_test_config_nlte.yml')
+        plasma_data['config'] = Configuration.from_yaml(config_path)
+        return plasma_data
+
+    #Additional Tests for NLTE Plasma
+
+    nlte_general_properties = ['beta_electron']
+    nlte_ion_population_properties = ['delta', 'previous_electron_densities']
+    nlte_radiative_properties = ['previous_beta_sobolev']
+    nlte_properties = nlte_general_properties + \
+        nlte_ion_population_properties + nlte_radiative_properties
+
+    @pytest.mark.parametrize("attr", nlte_properties)
+    def test_nlte_properties(self, plasma, reference, attr):
+        actual = getattr(plasma, attr)
+        if actual.ndim == 1:
+            actual = pd.Series(actual)
+        else:
+            actual = pd.DataFrame(actual)
+        expected = reference.select(os.path.join('plasma', attr))
+        pdt.assert_almost_equal(actual, expected)
+
+    def test_zeta_data(self, plasma, reference):
+        actual = getattr(plasma, 'zeta_data')
+        expected = reference.select(os.path.join('plasma', 'zeta_data'))
+        assert_almost_equal(actual, expected.values)
```

Comment:
@yeganer If I do `pdt.assert_almost_equal(actual, expected)`, I am getting this error
![image](https://user-images.githubusercontent.com/12808297/29442536-cf83c404-83ef-11e7-9c0e-752ee6404305.png)

I tried these options for changing `dtype` of columns for `actual` zeta data dataframe.
- `actual = actual.apply(pd.to_numeric, errors='coerce')`
- `actual = actual.convert_objects(convert_numeric=True)`
- `actual = actual.astype(np.int64)`

And, None of the above options are working for me, I want to convert `dtype` of `actual` zeta data attribute from `object` to `int64`, to make assertion possible here .